### PR TITLE
Add book

### PR DESCRIPTION
**Pull request by llbbdd**
User can add a book to the database

### CODE REVIEW

File: helpers.js [hunk 1]
```diff
@@ -41,3 +41,13 @@ exports.validPhotoFileExtensions = ["jpg", "png"];
 exports.menu = [
   { slug: '/action-name', title: 'Text', icon: 'icon-name', }
 ];
+
+exports.bookTagOptions = [
```
Comment (AllyMurray) [inline helpers.js]: Would we be better storing the book tag options in the database? We could then allow admin users to add new options. If you think this is a complication we don't need then we can leave it as is, it was just a thought.

File: models/Book.js [hunk 2]
```diff
@@ -84,17 +84,7 @@ module.exports = mongoose.model(model.Book, new mongoose.Schema({
     type: String,
     trim: true
   },
-  price: {
-    type: mongoose.Types.Currency,
-    required: "Please enter a valid price",
-    min: 1, // £0.01
-  },
-  tags: [String],
-  owner: {
```
Comment (AllyMurray) [inline models/Book.js]: Does the book no longer have an owner?

File: views/books.pug [hunk 3]
```diff
@@ -3,7 +3,8 @@ extends layout
 include mixins/_bookCard
```
Comment (AllyMurray) [inline views/books.pug]: Indentation looks wrong, I am not anal about this but it can cause issues in Pug files.

File: helpers.js (hunk 1 above)
Comment (llbbdd) [reply helpers.js]: Yes, that's a better idea. I'll set that up.

File: models/Book.js (hunk 2 above)
Comment (llbbdd) [reply models/Book.js]: I had 'owner' in twice. No idea why, or why Mongoose didn't throw an error.

Comment (AllyMurray) [reply models/Book.js]: Ah cool, that wasn't obvious in the diff.

**Pull request merged**